tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -48,9 +48,9 @@ const (
 {{/* TODO(ROX-10624): Remove this condition after all PKs fields were search tagged (PR #1653) */}}
 {{- if gt (len $pks) 1 }}
         existsStmt = "SELECT EXISTS(SELECT 1 FROM {{.Table}} WHERE {{template "whereMatch" $pks}})"
+        getStmt = "SELECT serialized FROM {{.Table}} WHERE {{template "whereMatch" $pks}}"
```

Comment:
What is the approach for SAC when obj has more than primary key?

Reply:
Until `ROX-10624` is solved we use the old apprach for multiple keys after this we will use exact matches